github/MainClass.py

```diff
@@ -300,6 +300,19 @@ def get_repos(self, since=github.GithubObject.NotSet):
             url_parameters
         )
 
+    def get_project(self, id):
```

Comment:
You can also get project columns and cards by their global ID, but I didn't add support for those.

Reply:
Okay, don't seem to have good use cases here. Future PR by someone else maybe.

Reply:
`MainClass.get_project` is a direct implementation of [this API endpoint](https://developer.github.com/v3/projects/#get-a-project).

The use case I see is if someone wants to record a project ID for future retrieval, which seems likely and reasonable. There is no other way to get to a project unless you list all projects for a repo or organization and then find the one you want - you cannot retrieve a single project from either a repo or organization, so I think this method is required. Let me know if you agree.

Reply:
Oops, sorry, just realized you were commenting on my comment, not my change. Never mind!

Reply:
Yup